text.md

```diff
@@ -0,0 +1 @@
+This file should be ignore by git
```

Comment:
- [ ] This file is required to be ignored by git so that it would not show on your repository. Kindly follow this [link](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) to add it to the .gitignore file.

### Project Requirement.

![ignore](https://github.com/MoliBravo/Hello-Microverse/assets/26597432/75f7f786-46ef-4356-8c5c-a51181a1eacf)